### ISSUE TITLE
python error

### ISSUE THREAD

**Issue by gioman** [alexbruy/statist#6]
QGIS 3.10 on Ubuntu 20.04

Spotted this python error when trying configure subplots

```
Traceback (most recent call last):
  File "C:\OSGEO4~1\apps\Python37\lib\site-packages\matplotlib\backends\backend_qt5.py", line 819, in configure_subplots
    dia = SubplotToolQt(self.canvas.figure, self.canvas.parent())
TypeError: 'NoneType' object is not callable
```

but have not been able to replicate in a consistent matter.

**Comment by gioman** [comment]
Sorry, I meant Windows 10.

**Comment by alexbruy** [comment]
Fixed in 3.2.
